source/community/reactnative/src/visualizer/renderer.js

```diff
@@ -145,7 +144,11 @@ export default class Renderer extends React.Component {
 
 const styles = StyleSheet.create({
     container: {
-        flex: 1,
+        ...Platform.select({
+            ios: {
+                marginBottom: 84,
```

Comment:
Reason for the value 84. Also move this to a const.

Reply:
Done.